src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -348,12 +348,17 @@ func (s *Service) sendToRooms(cli *gomatrix.Client, feedURL string, feed *gofeed
 	return nil
 }
 
-// SomeOne posted a new article: Title Of The Entry ( https://someurl.com/blag )
 func itemToHTML(feed *gofeed.Feed, item gofeed.Item) gomatrix.HTMLMessage {
-	return gomatrix.GetHTMLMessage("m.notice", fmt.Sprintf(
-		"<i>%s</i> posted a new article: %s ( %s )",
-		html.EscapeString(feed.Title), html.EscapeString(item.Title), html.EscapeString(item.Link),
-	))
+	return &gomatrix.HTMLMessage{
+		Body: fmt.Sprintf("%s: %s (%s)",
+			html.EscapeString(feed.Title), html.EscapeString(item.Title), html.EscapeString(item.Link)),
+		MsgType: "m.notice",
+		Format: "org.matrix.custom.html",
+		FormattedBody: fmt.Sprintf("<strong>%s</strong>:<br><a href=\"%s\"><strong>%s</strong></a>",
+			html.EscapeString(feed.Title), html.EscapeString(item.Link), html.EscapeString(item.Title)),
+			// FeedTitle:
+			// Title of the Entry, as a Link to It
```

Comment:
What does this mean exactly? "and a Link to it"*?

Reply:
Title of the entry, _as_ a link to it -- the title of the entry is a link to the entry (see screenshot from Riot)

Reply:
Changed in https://github.com/matrix-org/go-neb/pull/269/commits/b99b1b808fccddc7c2b2d54b01c845f133802e2a to clarify